GitUI/CommandsDialogs/SettingsDialog/Pages/AppearanceSettingsPage.Designer.cs

```diff
@@ -468,7 +468,7 @@ private void InitializeComponent()
             this.pictureAvatarHelp.Name = "pictureAvatarHelp";
             this.pictureAvatarHelp.Size = new System.Drawing.Size(16, 16);
             this.pictureAvatarHelp.SizeMode = System.Windows.Forms.PictureBoxSizeMode.AutoSize;
-            this.pictureAvatarHelp.TabIndex = 8;
+            this.pictureAvatarHelp.TabIndex = 3;
```

Comment:
Same index as AvatarProvider

Reply:
The help doesn't have a tab action and there's no TabStop on it, is thus inconsequential.

Reply:
Remove then?